### ISSUE TITLE
Add a feature that allows an admin to email users welcome email with their login info

### ISSUE THREAD

**Issue by MikeAlhayek** [OrchardCMS/OrchardCore#12111]
### Is your feature request related to a problem? Please describe.
When a new user is created from admin, there is no way to notify the user that their account was created.

### Describe the solution you'd like
It would be nice to add a checkbox in the create user screen to allow the user to email the new user their username/password and instruction on how to access the account. We would have to add a template in the settings so the user is able to change the body of the message along with common shortcodes. Or, create a view for the template so the user is able to override the message if needed.

**Comment by jptissot** [comment]
I think you can do this by using the `UserCreatedEvent` workflow event + an email task ?

**Comment by CrestApps** [comment]
@jptissot I don't think that is enough. When we create a new user, we also need to send them their temporary password in a plain text. The plain text password is not available in the event.

**Comment by hishamco** [comment]
If I'm wrong we already have `Users Registration` feature that allow users to sign up and confirm their emails

**Comment by CrestApps** [comment]
@hishamco that provides an email when the user is created from the front end. This request it to add a check box to the allow admin to send the newly created credentials via email when an admin adds a new user
